src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodoStatus } from '../../types/TodoStatus';
+
+type Props = {
+  visibleTodos: Todo[] | null,
+  status: string,
+  setStatus: (status: TodoStatus) => void,
+  areCompleted: Todo[] | null,
+  handleClearCompleted: () => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  visibleTodos,
+  status,
+  setStatus,
+  areCompleted,
+  handleClearCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {visibleTodos?.length !== 1
+          ? `${visibleTodos?.length} items left`
```

Comment:
So what if `visibleTodos` is `null`?

Reply:
@silchyn Should I create another condition here?

Reply:
That depends on the logic you're building. Should `Footer` be rendered if `visibleTodos === null`? If yes, what should the user see?